client/main.html

```diff
@@ -1,5 +1,7 @@
 <head>
   <title>Expense Report</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="./css/login.css" />
   <link rel="stylesheet" type="text/css" href="./css/dashboard.css" />
   <script type="text/babel" src="main.js"></script>
```

Comment:
I know we are not focusing on mobile devices, but it is helpful placing '<meta name="viewport" content="width=device-width, initial-scale=1.0">' in the <head> section to make more mobile friendly.